### PR TITLE
Add contact info to docs

### PR DESCRIPTION
**Pull request by mvidalgarcia**

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -96,3 +96,9 @@ Indices and tables
 .. |license| image:: https://img.shields.io/github/license/indico/indico.png
                    :alt: License
                    :target: https://github.com/indico/indico/blob/master/COPYING
+
+Contact
+++++++++++++++++++
```
Comment (ThiefMaster) [inline docs/source/index.rst]: that's way too many `+` chars... the underline should be just as long as the line before

File: CHANGES.rst [hunk 2]
```diff
@@ -56,6 +56,7 @@ Improvements
 - When blocking a user account, the user will be forcefully logged out in
   addition to being prevented from logging in
 - Warn when editing a speaker/author would result in duplicate emails
+- Add contact information to documentation (:issue:`3111`)
```
Comment (ThiefMaster) [inline CHANGES.rst]: IMO documentation changes aren't really useful in the changelog

File: docs/source/index.rst [hunk 3]
```diff
@@ -96,3 +96,9 @@ Indices and tables
...
+
+* IRC Channel: Official `#indico <https://webchat.freenode.net/?channels=indico>`_ channel at Freenode (irc.freenode.net). Also accessible through `Matrix <https://riot.im/app/#/room/#indico:matrix.org>`_.
+* Discussion forum: `Indico Community <https://talk.getindico.io>`_ Discourse instance
```
Comment (ThiefMaster) [inline docs/source/index.rst]: is the fact that it's a Discourse instance really relevant?

Comment (mvidalgarcia) [reply docs/source/index.rst]: Just copied it from https://getindico.io/contact/, but I agree it's no really relevant.

**Comment by ThiefMaster** [comment]
It would be nice if it showed up in the side menu (with the RTD theme) - otherwise people who are e.g. in the setup guide won't see it. Probably needs to be a separate page for that :/

**Comment by ThiefMaster** [comment]
http://fabric-www.readthedocs.io/contact.html looks pretty nice for example

**Pull request merged**